kore.maude

```diff
@@ -220,18 +227,30 @@ Questions about this definition:
 - do we really need ~> as a K construct?  Can it be a label and thus the way K
   deal with evaluation contexts just a particular methodology and not an
   intrisic part of K ? Same question for =>.
+  Cosmin: Syntacticly at least, I would say "no" for both. From the implementation
+  perspective, this will simplify traversal code even further. 
 - rename .::K and .::KList to . ?  if we get rid of ~> as above, then we may
   only need .::KList.
+  Cosmin: I agree.
 - Should we allow empty productions instead of sort declarations?  A pro
   argument is that we have fewer syntactic constructs.  A cons argument is
   that a sort declaration may be regarded as a different entityr, with
   different kinds of attributes.
+  Cosmin: I don't quite follow.
 - Should we add constant AttributeKeys which have special semantics, like
   "strict", as constants?
+  Cosmin: we may want to discuss the attribute syntax separately.
 - Brackets and disappearing injections are the same thing, arent's they?  A
   production which only has one non-terminal and the production disappears
   from the AST.
+  Cosmin: Yes, I also think they can be seen as the same thing. That being
+          said, I would really like for injections to not dissapear. If something
+          is meant to dissapear, it should be declared as a subsort. And the
+          dissapearing injections (including subsorts) could be done via rules as
+          we discussed before.
 - Should #Label contain more structured information than just a String?
+  Cosmin: From my perspective, its semantics and any other information should live
+          with the definition (module), not within the label.
 
```

Comment:
So the answer is NO?  I was thinking more about implementing kompile using Maude and this definition of KORE.  In that case, instead of having a table holding for each KLabel where it comes from, including its argument sorts and its result sort, all that information could have already been part of the KLabel term.  Or are you saying that we may only need to add the module name info to the KLabel term, and then we will be able to get the info from there?